### DISCUSSION TITLE
User specific UI

### DISCUSSION THREAD

**Discussion by brian-liegl-sada** [drakkan/sftpgo#279] [Ideas]
Is there anything in the works for doing a user specific user management UI that would allow a registered user to access their credentials and allow them to change only their password or add public keys.  We should not give them access to review any other user information, or the storage details.

**Comment by drakkan** [comment]
Hi,

for now I'm refactoring the admin interface, see #197. This is the first step before we can enable an interface for users too. 

At some point I would like to add the user interface you suggest maybe allowing users to also browse files and share them, something similar to the MinIO user interface. But this development is not imminent, I don't have a precise schedule depends on my free time

**Reply by brian-liegl-sada** [reply]
Thanks for the update, please keep me posted.

**Reply by brian-liegl-sada** [reply]
Just wondering how the refactoring is going of the admin page.

**Reply by drakkan** [reply]
The refactoring is done, please give a try to git master. 
I'm finalyzing v2.0.0 now, see #287. The user interface will probably be added after 2.0.0 otherwise if I never stop development new versions will never be released. More users should use the latest development version :)

**Comment by brian-liegl-sada** [comment]
Thanks for the quick response on this, do you have a potential timeline for
the post 2.0.0 release?

On Fri, Jan 29, 2021 at 12:23 PM Nicola Murino <notifications@github.com>
wrote:

> The refactoring is done, please give a try to git master.
> I'm finalyzing v2.0.0 now, see #287
> <https://github.com/drakkan/sftpgo/issues/287>. The user interface will
> probably be added after 2.0.0 otherwise if I never stop development new
> versions will never be released. More users should use the latest
> development version :)
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/drakkan/sftpgo/discussions/279#discussioncomment-321561>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AR6H2S4DTHBN4J3XBH2OZD3S4L4KTANCNFSM4WCXIO4Q>
> .
>


-- 
[image: SADA Systems is a premier cloud solutions provider specializing in
technology consulting, IT services, application development, and managed
services.] <https://sada.com/?utm_source=emailSig>
[image: Twitter] <https://sada.com/lnk/twitter>[image: Facebook]
<https://sada.com/lnk/facebook>[image: LinkedIn]
<https://sada.com/lnk/linkedin>[image: Youtube]
<https://sada.com/lnk/youtube>

BRIAN LIEGL

Data Engineer

O: (920) 241-3859 <+19202413859>

**Reply by drakkan** [reply]
I don't have a definite schedule, releases are feature-driven for now. A new version generally takes a few months. If you want to prioritize your issues/requests please consider a [sponsorship](https://github.com/sponsors/drakkan). Thank you!
